Src/Plugins/Security/Security.SharedStrings.txt

```diff
@@ -44,7 +44,7 @@
   $SEC101/015.GoogleGCMServiceAccount=(("|'|`)?type("|'|`)?\s{0,50}(:|=>|=)\s{0,50}("|'|`)?service_account("|'|`)?,?)
   $SEC101/016.StripeApiKey=\b(?<refine>(?<key>(?:r|s)k_(live|test)_(?i)[0-9a-z]{24,99}))([^0-9a-z]|$)
   $SEC101/017.NpmAuthorToken=(?i)npm.{0,100}[^0-9A-Za-z](?-i)(?<refine>(?<key>[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}))[^0-9A-Za-z]
-  $SEC101/019.PicaticApiKey=\b(?<refine>sk_(live|test)_[0-9a-z]{32})
+  $SEC101/019.PicaticApiKey=\b(?<refine>sk_(live|test)_[0-9a-z]{32})[^0-9A-Za-z]
```

Comment:
>[^0-9A-Za-z] [](start = 68, length = 12)

(?i)([^0-9a-z]|$)

Reply:
thanks for reviewing. already created a new pr and merged.

---
In reply to: [591968421](https://github.com/microsoft/sarif-pattern-matcher/pull/283#discussion_r591968421) [](ancestors = 591968421)